src/main/kotlin/com/openlattice/transporter/types/TransporterColumnSet.kt

```diff
@@ -0,0 +1,31 @@
+package com.openlattice.transporter.types
+
+import com.openlattice.ApiUtil
+import com.openlattice.edm.PostgresEdmTypeConverter
+import com.openlattice.edm.type.PropertyType
+import com.openlattice.postgres.PostgresColumnDefinition
+import com.openlattice.postgres.PostgresDataTables
+import com.openlattice.postgres.PostgresDatatype
+import java.util.*
+
+data class TransporterColumn(val srcCol: String, val destColName: String, val dataType: PostgresDatatype) {
```

Comment:
Can you please add some javadoc for the variables and the `destCol()` function? (Also for `TransporterColumnSet `)

Reply:
I renamed them, because they were definitely confusing. I can add some docs if they're still confusing.